README.md

```diff
@@ -89,6 +89,29 @@ This will send a dummy task to the ServiceManager. Make sure to have the Service
 
 The plonk proof is located at `task_sender/test_examples`.
 
+## Deployment
+
+To deploy the contracts to Testnet/Mainnet, you will need to set environment variables
+in a .env file in the same directory as the deployment script (`contracts/scripts/`). 
```

Comment:
Specify here that you need to set addresses on `contracts/script/deploy/config/holesky/aligned.holesky.config.json` as well